test/api/package/npm/delete-ownership.test.js

```diff
@@ -101,7 +118,34 @@ test('DELETE `/package/npm/ownership` 200 success', async (t) => {
 
   // the package they were co-maintaining is now fully the other person's
   const sodium = await t.context.db.package.get({ packageId: t.context.sodium })
-  t.deepEqual(sodium.maintainers, [{ userId: t.context.userId2, revenuePercent: 100 }])
+  t.deepEqual(sodium.maintainers, [
+    { userId: t.context.userId2, revenuePercent: 60, source: 'registry' },
+    { userId: t.context.userId3, revenuePercent: 10, source: 'invite' }
+  ])
+})
+
+test('DELETE `/package/npm/ownership` 200 success | remove only registry sources', async (t) => {
+  const { userId3: userId } = t.context
+
+  const res = await t.context.app.inject({
+    method: 'DELETE',
+    url: '/package/npm/ownership',
+    payload: {},
+    headers: {
+      cookie: `${USER_WEB_SESSION_COOKIE}=${t.context.session3}`
+    }
+  })
+  t.deepEqual(res.statusCode, 200)
+
+  // the db now shows they maintain 1 javascript packages, the one where the source is invite
+  const jsPackages = await t.context.db.package.getOwnedPackages({ userId, registry: REGISTRIES.NPM, language: LANGUAGES.JAVASCRIPT })
+  t.is(jsPackages.length, 1)
+
+  // the package they were co-maintaining still shows user 3 as a maintainer because they were source 'invite'
+  // deleting ownership will not touch packages where the source is invite, only where source
+  // is registry
+  const sodium = await t.context.db.package.get({ packageId: t.context.sodium })
+  t.true(!!sodium.maintainers.find(m => (m.userId === t.context.userId3 && m.revenuePercent === 10 && m.source === 'invite')))
```

Comment:
nit: instead of `t.true(!!array.find())` this might be a good use case for `t.true(array.some())`